scripts/performance.ps1

```diff
@@ -271,7 +271,7 @@ $RemotePlatform = Invoke-TestCommand -Session $Session -ScriptBlock {
     }
 }
 
-$OutputDir = Join-Path $RootDir "artifacts/PerfDataResults/$RemotePlatform/$($RemoteArch)_$($Config)_$($RemoteTls)"
+$OutputDir = Join-Path $RootDir "artifacts/PerfDataResults/$RemotePlatform/$($RemoteArch)_$($Config)_$($RemoteTls)$($ExtraArtifactDir)"
```

Comment:
FYI, this makes it so several PGO runs can run in parallel and not overwrite each other.

Reply:
That is very handy! I'm gonna include that in my PR.